web/annotation_editor_layer_builder.css

```diff
@@ -167,3 +168,37 @@
   height: 100%;
   touch-action: none;
 }
+
+.annotationEditorLayer .stampEditor {
+  position: absolute;
+  background: transparent;
+  border-radius: 3px;
+  overflow: auto;
+  width: auto;
+  height: auto;
+  z-index: 1;
+  transform-origin: 0 0;
+  cursor: auto;
+}
```

Comment:
Could we perhaps reduce some duplication here, by [combing this existing code](https://github.com/mozilla/pdf.js/blob/85b1d35eefae74b578efc5957cd56c9cfc7d9d2b/web/annotation_editor_layer_builder.css#L84-L96) with the new one along these lines?
```css
.annotationEditorLayer :is(.freeTextEditor, .stampEditor)  {
  position: absolute;
  background: transparent;
  border-radius: 3px;
  width: auto;
  height: auto;
  z-index: 1;
  transform-origin: 0 0;
  cursor: auto;
}
.annotationEditorLayer .freeTextEditor {
  padding: calc(var(--freetext-padding) * var(--scale-factor));
  resize: none;
  touch-action: none;
}
.annotationEditorLayer .stampEditor  {
  overflow: auto;
}
```